packrat/src/main/java/com/github/reflectoring/infiniboard/packrat/dashboard/DashboardConfig.java

```diff
@@ -1,16 +1,23 @@
 package com.github.reflectoring.infiniboard.packrat.dashboard;
 
+import java.util.List;
+
 import org.springframework.data.annotation.Id;
 
+import com.github.reflectoring.infiniboard.packrat.widget.WidgetConfig;
+
 public class DashboardConfig {
 
     @Id
     private String id;
 
     private String title;
 
-    public DashboardConfig(String title) {
-        this.title = title;
+    private String description;
+
+    private List<WidgetConfig> widgetConfigs;
+
+    public DashboardConfig() {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1186)